packages/hooks/package.json

```diff
@@ -1,9 +1,9 @@
 {
     "name": "@cura/hooks",
-    "version": "0.0.6",
+    "version": "0.0.8",
```

Comment:
Shouldn't `hooks` version be updated in the frontend too?https://github.com/curaOS/source/blob/88642b35c998f1309e2f7931693c6ad2dc9b613c/examples/frontend/package.json#L12